### PR TITLE
Added a local module to generate bed files with variant caller annotations

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

A groovy function was written to generate this file before, but it messed up nextflow's caching feature like cnvnator did before. So now I have switched out this function in favor of a local module.

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Installed the nf-core version of ensemblvep/vep module [#482](https://github.com/nf-core/raredisease/pull/482)
 - The filenames of the ranked output VCF files have been changed. See [output.md](docs/output.md#filtering-and-ranking) for more information[#485](https://github.com/nf-core/raredisease/pull/485)
 - Patched cnvnator module so that the processes didn't have to rerun after a failed run [#503](https://github.com/nf-core/raredisease/pull/503).
+- Added a local module to generate bed files with variant caller names [#503](https://github.com/nf-core/raredisease/pull/505).
```
Comment (jemten) [inline CHANGELOG.md:51]:
```suggestion
- Added a local module to generate bed files with variant caller names [#505](https://github.com/nf-core/raredisease/pull/505).
```

File: modules/local/add_varcallername_to_bed.nf [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/tabix:1.11--hdfd78af_0' :
+        'biocontainers/tabix:1.11--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(chromsizes)
+
+    output:
+    tuple val(meta), path("*.gz"), path("*.tbi"), optional: true, emit: gz_tbi
```
Comment (jemten) [inline modules/local/add_varcallername_to_bed.nf:14]:
```suggestion
    tuple val(meta), path("*.gz"), path("*.tbi"), emit: gz_tbi
```
Does it need to be optional?

**Review by jemten — COMMENTED** [review]
Two small comments, but otherwise it looks good 👍

File: modules/local/add_varcallername_to_bed.nf (hunk 2 above)
Comment (ramprasadn) [reply modules/local/add_varcallername_to_bed.nf]: I guess not since tbi index will always be generated alongside the bed file

**Review by jemten — APPROVED** [review]
👍

**Pull request merged**